_ext/link_renderer.rb

```diff
@@ -0,0 +1,59 @@
+##
+#
+# An extension which renders the post links using the Twitter Bootstrap pagination styles. Requires the default
+# pagination extension to be enabled.
+#
+##
+module InRelationTo
+  module Extensions
+    class PaginationLinkRenderer
+
+      module BootstrapPaginationLinkRenderer
+
+        def bootstrap_links
```

Comment:
I took the original links method as an example and adapted it to use the Bootstrap CSS classes.